st/srcery_st.h

```diff
@@ -0,0 +1,38 @@
+/* Terminal colors (16 first used in escape sequence) */
+static const char *colorname[] = {
+	/* 8 normal colors */
+	"#1c1b19",
+	"#ef2f27",
+	"#519f50",
+	"#fbb829",
+	"#2c78bf",
+	"#e02c6d",
+	"#0aaeb3",
+	"#d0bfa1",
+
+	/* 8 bright colors */
+	"#918175",
+	"#f75341",
+	"#98bc37",
+	"#fed06e",
+	"#68a8e4",
+	"#ff5c8f",
+	"#53fde9",
+	"#fce8c3",
+
+	[255] = 0,
+
+	/* more colors can be added after 255 to use with DefaultXX */
+	"#fce8c3", /* foreground */
+	"#1c1b19", /* background */
+	"#fbb829", /* cursor */
+};
+
+/*
+ * Default colors (colorname index)
+ * foreground, background, cursor, reverse cursor
+ */
+unsigned int defaultfg = 256;
```

Comment:
For me, that is not fluent with C, care to explain why are some static, and some not?

Reply:
Here is the reference:
https://git.suckless.org/st/file/config.def.h.html#l130

Reply:
Thank you. 😄